Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1094,13 +1094,15 @@ theorem bfamilyOfFamily_typein {ι} (f : ι → α) (i) :
     bfamilyOfFamily f (typein _ i) (typein_lt_type _ i) = f i :=
   bfamilyOfFamily'_typein _ f i
 
-@[simp, nolint simpNF] -- Porting note (#10959): simp cannot prove this
+-- See https://github.com/leanprover-community/batteries/issues/365 for the `nolint simpNF`.
+@[simp, nolint simpNF]
```

Comment:
If this isn't being used as a `simp` lemma, I think it's fine to remove this outright. This `BFamily` API has no reason to be used in any new theorems and is planned to be deprecated soon.